cvat/schema.yml

```diff
@@ -7140,6 +7866,66 @@ components:
             type: string
             minLength: 1
           writeOnly: true
+    PatchedQualitySettingsRequest:
+      type: object
+      properties:
+        iou_threshold:
+          type: number
+          format: double
+          description: Used for distinction between matched / unmatched shapes
+        oks_sigma:
+          type: number
+          format: double
+          description: |
+            Like IoU threshold, but for points.
+            The percent of the bbox area, used as the radius of the circle around the GT point,
+            where the checked point is expected to be.
+            Read more: https://cocodataset.org/#keypoints-eval
+        line_thickness:
+          type: number
+          format: double
+          description: |
+            Thickness of polylines, relatively to the (image area) ^ 0.5.
+            The distance to the boundary around the GT line,
+            inside of which the checked line points should be
+        low_overlap_threshold:
+          type: number
+          format: double
+          description: Used for distinction between strong / weak (low_overlap) matches
+        oriented_lines:
+          type: boolean
+          description: Indicates that polylines have direction
+        line_orientation_threshold:
+          type: number
+          format: double
+          description: |
+            The minimal gain in the GT IoU between the given and reversed line directions
+            to consider the line inverted. Only useful with the 'oriented_lines' parameter
+        compare_groups:
+          type: boolean
+          description: Enables or disables annotation group checks
+        group_match_threshold:
+          type: number
+          format: double
+          description: |
+            Minimal IoU for groups to be considered matching,
+            used when the 'compare_groups' is enabled
+        check_covered_annotations:
+          type: boolean
+          description: Check for partially-covered annotations, useful in segmentation
+            tasks
+        object_visibility_threshold:
+          type: number
+          format: double
+          description: |
+            Minimal visible area percent of the spatial annotations (polygons, masks)
+            for reporting covered annotations,
+            useful with the 'check_covered_annotations' option
+        panoptic_comparison:
+          type: boolean
+          description: Use only the visible part of the masks and polygons in comparisons
+        compare_attributes:
+          type: boolean
```

Comment:
Description?

Reply:
What would you like to explain in the description?

Reply:
Added